recipes/appimage-updater-bridge/meta.yaml

```diff
@@ -31,8 +31,11 @@ requirements:
     - qt
 
 test:
+  source_files:
+    - tests/*
   commands:
-    - test -f ${PREFIX}/plugins/libAppImageUpdaterBridge.so  # [unix]
+    - test -f ${PREFIX}/plugins/libAppImageUpdaterBridge.so
+    - bash start_test.sh
```

Comment:
Could we try running the lines from the script directly?

Reply:
Also the qmake file in test depends on AppImageUpdaterBridge.pri file. It includes it. ```include(.../AppImageUpdaterBridge.pri)``` does it cause some sort of problem?

The ```AppImageUpdaterBridge.pri``` is in the root of the repo FYI.

Reply:
we need to run it in the build step anyway as  compilers are needed.
But I guess qmake is less supported compared to cmake. So it definitely helps if you add tests to cmake.

Reply:
Okay I will look into it.

Reply:
@looooo I can write a CMakeLists for the test to compile but do you need that file in the repo itself in a new version? or can you somehow just echo the file into place? Also in order to compile with CMake you need to install the library with cmake to the system and then compile the tests with CMake. ie.,

```
 $ cd AppImageUpdaterBridge
 $ cmake -DNO_GUI=ON -DNO_LOGGING .
 $ make -j$(nproc)
 $ sudo make install
 $ cd tests
 $ # Echo the CMakeList.txt
 $ cmake .
 $ make -j$(nproc)
 $ # Download the test appimages with wget
 $ ./AppImageUpdaterBridgeTests # Eval the test.
```

Reply:
best would be a little application that runs without additional compilation.

what about a python script which checks the availability of the plugin?

Reply:
Ah that is already present in the examples directory. ```examples/PyQt5Update```. But it requires an testing AppImage to do the update.